Tests/Unit/EmogrifierTest.php

```diff
@@ -948,6 +948,41 @@ public function emogrifyAppliesCssFromStyleNodes()
     /**
      * @test
      */
+    public function emogrifyDoesNotApplyCssFromStyleBlocksWhenDisabled()
```

Comment:
Could you please move the precondition (WhenDisabled) to before the result (DoesNotApply...)?

And if you like, you can make the wording shorter by using a bit of pidgin English: DoesNotApply -> NotApplies

Ditto for the other test name.